12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.API ( 
	API_ID               integer GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           identity   NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	URL                  varchar(255)   NOT NULL,
	SECRET_KEY           varchar(255)   NOT NULL,
	BUCKET               varchar(255)   ,
	SUBTENANT            varchar(255)   ,
	HTTP_ADDRESS_IP      varchar(16)   NOT NULL,
	HTTPS_ADDRESS_IP     varchar(16)   ,
	HTTP_ADDRESS_PORT    varchar(5)   NOT NULL,
	HTTPS_ADDRESS_PORT   varchar(5)   ,
	CONSTRAINT PK_API PRIMARY KEY ( API_ID ),
	CONSTRAINT PK_API_0 UNIQUE ( ACCOUNT_ID ) ,
	CONSTRAINT PK_API_1 UNIQUE ( API_TYPE_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.API IS 'API information' [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.API_SELECTED ( 
	API_SELECTED_ID      identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	TESTCONFIGURATION_ID identity   NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	API_SELECTED_HTTP    boolean   ,
	API_SELECTED_HTTPS   boolean   ,
	CONSTRAINT PK_PROTOCOL PRIMARY KEY ( API_SELECTED_ID ),
	CONSTRAINT PK_API_SELECTED UNIQUE ( API_TYPE_ID ) ,
	CONSTRAINT PK_API_SELECTED_0 UNIQUE ( TESTCONFIGURATION_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API_SELECTED" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.API_SELECTED IS 'The API(s) selected in the Test Plan. These will be the ones to be tested.' [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.EXECUTION_PLAN ( 
	EXECUTION_PLAN_ID    identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           identity   NOT NULL,
	CONFIGURATION_TEMPLATE_ID identity   ,
	TESTCONFIGURATION_ID identity   ,
	EXECUTION_REPORT_ID  identity   NOT NULL,
	CONSTRAINT PK_EXECUTIONPLAN PRIMARY KEY ( EXECUTION_PLAN_ID ),
	CONSTRAINT PK_EXECUTIONPLAN_0 UNIQUE ( ACCOUNT_ID ) ,
	CONSTRAINT PK_EXECUTIONPLAN_1 UNIQUE ( CONFIGURATION_TEMPLATE_ID ) ,
	CONSTRAINT PK_EXECUTIONPLAN_2 UNIQUE ( TESTCONFIGURATION_ID ) ,
	CONSTRAINT PK_EXECUTION_PLAN UNIQUE ( EXECUTION_REPORT_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_PLAN" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.EXECUTION_PLAN IS 'Execution plant junction table maps the the Test configurations done for each one of the tests, and their test results. ' [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.EXECUTION_REPORT_DATA ( 
	EXECUTION_REPORT_DATA_ID identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	EXECUTION_REPORT_ID  identity   NOT NULL,
	DATA_KEY             varchar(255)   NOT NULL,
	DATA_VALUE           varchar(500)   NOT NULL,
	CONSTRAINT PK_EXECUTIONREPORTDATA PRIMARY KEY ( EXECUTION_REPORT_DATA_ID ),
	CONSTRAINT PK_EXECUTION_REPORT_DATA UNIQUE ( EXECUTION_REPORT_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_REPORT_DATA" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.EXECUTION_REPORT_DATA IS 'This table holds a data row for each data item in the report. It has the form of a dictionanary Key-Value pairs' [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.TEST_CONFIGURATION ( 
	TESTCONFIGURATION_ID identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	OBJECT_SIZE_REPORT_UNIT_ID integer   NOT NULL,
	TEST_CONFIG_NAME     varchar(255)   NOT NULL,
	TEST_CONFIG_DESCRIPTION varchar(255)   ,
	OBJECT_SIZE          integer   ,
	NUMBER_OF_OPERATIONS integer   NOT NULL,
	NUMBER_OF_THREADS    integer  DEFAULT 1 NOT NULL,
	CREATE_OPERATION     boolean   ,
	READ_OPERATION       boolean   ,
	UPDATE_OPERATION     boolean   ,
	DELETE_OPERATION     boolean   ,
	CREATE_PERCENT       integer   ,
	READ_PERCENT         integer   ,
	UPDATE_PERCENT       integer   ,
	DELETE_PERCENT       integer   ,
	NUMBER_OF_RETRY      integer   ,
	ENABLE_RAND_SIZE_OBJECT boolean   ,
	MIN_RAND_SIZE_OBJECT integer   ,
	MAX_RAN_DSIZE_OBJECT integer   ,
	CREATED_DATE         date  DEFAULT CURRENT_DATE ,
	SMALL_SIZE_PERCENT   integer   ,
	MEDIUM_SIZE_PERCENT  integer   ,
	LARGE_SIZE_PERCENT   integer   ,
	API_SELCTED_ID       identity   NOT NULL,
	CONSTRAINT PK_CONFIGURATIONTEMPLATE_1 PRIMARY KEY ( TESTCONFIGURATION_ID ),
	CONSTRAINT PK_TEST_CONFIGURATION UNIQUE ( OBJECT_SIZE_REPORT_UNIT_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_CONFIGURATION" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.TEST_CONFIGURATION IS 'Test Configuration stores the configuration for the test to be executed.' [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.TEST_PLAN_OPERATION ( 
	TEST_PLAN_OPERATION_ID identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	CRUD_OPERATION_TYPE_ID identity   NOT NULL,
	CRUD_OPERATION       integer   NOT NULL,
	FILE_URL             varchar(255)   ,
	CONSTRAINT PK_TEST_PLAN_OPERATION PRIMARY KEY ( TEST_PLAN_OPERATION_ID ),
	CONSTRAINT PK_TEST_PLAN_OPERATION_0 UNIQUE ( API_TYPE_ID ) ,
	CONSTRAINT PK_TEST_PLAN_OPERATION_1 UNIQUE ( CRUD_OPERATION_TYPE_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_PLAN_OPERATION" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.TEST_PLAN_OPERATION IS 'This table contains the test Plan operations to be executed by the test plan. ' [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API" not found; SQL statement:
ALTER TABLE VSTRIKERDB.ACCOUNT ADD CONSTRAINT FK_ACCOUNT_API FOREIGN KEY ( ACCOUNT_ID ) REFERENCES VSTRIKERDB.API( ACCOUNT_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_PLAN" not found; SQL statement:
ALTER TABLE VSTRIKERDB.ACCOUNT ADD CONSTRAINT FK_ACCOUNT_EXECUTIONPLAN FOREIGN KEY ( ACCOUNT_ID ) REFERENCES VSTRIKERDB.EXECUTION_PLAN( ACCOUNT_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API" not found; SQL statement:
ALTER TABLE VSTRIKERDB.API_TYPE ADD CONSTRAINT FK_API_TYPE_API_TYPE_ID FOREIGN KEY ( API_TYPE_ID ) REFERENCES VSTRIKERDB.API( API_TYPE_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_PLAN_OPERATION" not found; SQL statement:
ALTER TABLE VSTRIKERDB.API_TYPE ADD CONSTRAINT FK_API_TYPE_TEST_PLAN_OPERATION FOREIGN KEY ( API_TYPE_ID ) REFERENCES VSTRIKERDB.TEST_PLAN_OPERATION( API_TYPE_ID ) ON DELETE NO ACTION ON UPDATE NO ACTION [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API_SELECTED" not found; SQL statement:
ALTER TABLE VSTRIKERDB.API_TYPE ADD CONSTRAINT FK_API_TYPE_API_SELECTED FOREIGN KEY ( API_TYPE_ID ) REFERENCES VSTRIKERDB.API_SELECTED( API_TYPE_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_PLAN" not found; SQL statement:
ALTER TABLE VSTRIKERDB.CONFIGURATION_TEMPLATE ADD CONSTRAINT FK_CONFIGURATION_TEMPLATE_EXECUTION_PLAN FOREIGN KEY ( CONFIGURATION_TEMPLATE_ID ) REFERENCES VSTRIKERDB.EXECUTION_PLAN( CONFIGURATION_TEMPLATE_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_PLAN_OPERATION" not found; SQL statement:
ALTER TABLE VSTRIKERDB.CRUD_OPERATION_TYPE ADD CONSTRAINT FK_CRUD_OPERATION_TYPE_TEST_PLAN_OPERATION FOREIGN KEY ( CRUD_OPERATION_TYPE_ID ) REFERENCES VSTRIKERDB.TEST_PLAN_OPERATION( CRUD_OPERATION_TYPE_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:21:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_REPORT_DATA" not found; SQL statement:
ALTER TABLE VSTRIKERDB.EXECUTION_REPORT ADD CONSTRAINT FK_EXECUTION_REPORT_EXECUTION_REPORT_DATA FOREIGN KEY ( EXECUTION_REPORT_ID ) REFERENCES VSTRIKERDB.EXECUTION_REPORT_DATA( EXECUTION_REPORT_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:21:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_PLAN" not found; SQL statement:
ALTER TABLE VSTRIKERDB.EXECUTION_REPORT ADD CONSTRAINT FK_EXECUTION_REPORT_EXECUTION_REPORT FOREIGN KEY ( EXECUTION_REPORT_ID ) REFERENCES VSTRIKERDB.EXECUTION_PLAN( EXECUTION_REPORT_ID ) ON DELETE NO ACTION ON UPDATE NO ACTION [42102-184]
12-19 15:21:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_CONFIGURATION" not found; SQL statement:
ALTER TABLE VSTRIKERDB.OBJECT_SIZE_REPORT_UNIT ADD CONSTRAINT FK_OBJECT_SIZE_REPORT_UNIT_TEST_CONFIGURATION FOREIGN KEY ( OBJECT_SIZE_REPORT_UNIT_ID ) REFERENCES VSTRIKERDB.TEST_CONFIGURATION( OBJECT_SIZE_REPORT_UNIT_ID ) ON DELETE NO ACTION ON UPDATE NO ACTION [42102-184]
12-19 15:21:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_CONFIGURATION" not found; SQL statement:
ALTER TABLE VSTRIKERDB.TEST_CONFIGURATION ADD CONSTRAINT FK_TEST_CONFIGURATION_EXECUTION_PLAN FOREIGN KEY ( TESTCONFIGURATION_ID ) REFERENCES VSTRIKERDB.EXECUTION_PLAN( TESTCONFIGURATION_ID ) ON DELETE NO ACTION ON UPDATE NO ACTION [42102-184]
12-19 15:21:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_CONFIGURATION" not found; SQL statement:
ALTER TABLE VSTRIKERDB.TEST_CONFIGURATION ADD CONSTRAINT FK_TEST_CONFIGURATION_TEST_CONFIGURATION_ID FOREIGN KEY ( TESTCONFIGURATION_ID ) REFERENCES VSTRIKERDB.API_SELECTED( TESTCONFIGURATION_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "VSTRIKERDB" already exists; SQL statement:
CREATE SCHEMA VSTRIKERDB [90078-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.CreateSchema.update(CreateSchema.java:48)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.API ( 
	API_ID               integer GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           identity   NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	URL                  varchar(255)   NOT NULL,
	SECRET_KEY           varchar(255)   NOT NULL,
	BUCKET               varchar(255)   ,
	SUBTENANT            varchar(255)   ,
	HTTP_ADDRESS_IP      varchar(16)   NOT NULL,
	HTTPS_ADDRESS_IP     varchar(16)   ,
	HTTP_ADDRESS_PORT    varchar(5)   NOT NULL,
	HTTPS_ADDRESS_PORT   varchar(5)   ,
	CONSTRAINT PK_API PRIMARY KEY ( API_ID ),
	CONSTRAINT PK_API_0 UNIQUE ( ACCOUNT_ID ) ,
	CONSTRAINT PK_API_1 UNIQUE ( API_TYPE_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.API IS 'API information' [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.API_SELECTED ( 
	API_SELECTED_ID      identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	TESTCONFIGURATION_ID identity   NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	API_SELECTED_HTTP    boolean   ,
	API_SELECTED_HTTPS   boolean   ,
	CONSTRAINT PK_PROTOCOL PRIMARY KEY ( API_SELECTED_ID ),
	CONSTRAINT PK_API_SELECTED UNIQUE ( API_TYPE_ID ) ,
	CONSTRAINT PK_API_SELECTED_0 UNIQUE ( TESTCONFIGURATION_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API_SELECTED" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.API_SELECTED IS 'The API(s) selected in the Test Plan. These will be the ones to be tested.' [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.EXECUTION_PLAN ( 
	EXECUTION_PLAN_ID    identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           identity   NOT NULL,
	CONFIGURATION_TEMPLATE_ID identity   ,
	TESTCONFIGURATION_ID identity   ,
	EXECUTION_REPORT_ID  identity   NOT NULL,
	CONSTRAINT PK_EXECUTIONPLAN PRIMARY KEY ( EXECUTION_PLAN_ID ),
	CONSTRAINT PK_EXECUTIONPLAN_0 UNIQUE ( ACCOUNT_ID ) ,
	CONSTRAINT PK_EXECUTIONPLAN_1 UNIQUE ( CONFIGURATION_TEMPLATE_ID ) ,
	CONSTRAINT PK_EXECUTIONPLAN_2 UNIQUE ( TESTCONFIGURATION_ID ) ,
	CONSTRAINT PK_EXECUTION_PLAN UNIQUE ( EXECUTION_REPORT_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_PLAN" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.EXECUTION_PLAN IS 'Execution plant junction table maps the the Test configurations done for each one of the tests, and their test results. ' [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.EXECUTION_REPORT_DATA ( 
	EXECUTION_REPORT_DATA_ID identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	EXECUTION_REPORT_ID  identity   NOT NULL,
	DATA_KEY             varchar(255)   NOT NULL,
	DATA_VALUE           varchar(500)   NOT NULL,
	CONSTRAINT PK_EXECUTIONREPORTDATA PRIMARY KEY ( EXECUTION_REPORT_DATA_ID ),
	CONSTRAINT PK_EXECUTION_REPORT_DATA UNIQUE ( EXECUTION_REPORT_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_REPORT_DATA" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.EXECUTION_REPORT_DATA IS 'This table holds a data row for each data item in the report. It has the form of a dictionanary Key-Value pairs' [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TABLE_0" already exists; SQL statement:
CREATE TABLE VSTRIKERDB.TABLE_0 ( 
 ) [42101-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.TEST_CONFIGURATION ( 
	TESTCONFIGURATION_ID identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	OBJECT_SIZE_REPORT_UNIT_ID integer   NOT NULL,
	TEST_CONFIG_NAME     varchar(255)   NOT NULL,
	TEST_CONFIG_DESCRIPTION varchar(255)   ,
	OBJECT_SIZE          integer   ,
	NUMBER_OF_OPERATIONS integer   NOT NULL,
	NUMBER_OF_THREADS    integer  DEFAULT 1 NOT NULL,
	CREATE_OPERATION     boolean   ,
	READ_OPERATION       boolean   ,
	UPDATE_OPERATION     boolean   ,
	DELETE_OPERATION     boolean   ,
	CREATE_PERCENT       integer   ,
	READ_PERCENT         integer   ,
	UPDATE_PERCENT       integer   ,
	DELETE_PERCENT       integer   ,
	NUMBER_OF_RETRY      integer   ,
	ENABLE_RAND_SIZE_OBJECT boolean   ,
	MIN_RAND_SIZE_OBJECT integer   ,
	MAX_RAN_DSIZE_OBJECT integer   ,
	CREATED_DATE         date  DEFAULT CURRENT_DATE ,
	SMALL_SIZE_PERCENT   integer   ,
	MEDIUM_SIZE_PERCENT  integer   ,
	LARGE_SIZE_PERCENT   integer   ,
	API_SELCTED_ID       identity   NOT NULL,
	CONSTRAINT PK_CONFIGURATIONTEMPLATE_1 PRIMARY KEY ( TESTCONFIGURATION_ID ),
	CONSTRAINT PK_TEST_CONFIGURATION UNIQUE ( OBJECT_SIZE_REPORT_UNIT_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_CONFIGURATION" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.TEST_CONFIGURATION IS 'Test Configuration stores the configuration for the test to be executed.' [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.TEST_PLAN_OPERATION ( 
	TEST_PLAN_OPERATION_ID identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	CRUD_OPERATION_TYPE_ID identity   NOT NULL,
	CRUD_OPERATION       integer   NOT NULL,
	FILE_URL             varchar(255)   ,
	CONSTRAINT PK_TEST_PLAN_OPERATION PRIMARY KEY ( TEST_PLAN_OPERATION_ID ),
	CONSTRAINT PK_TEST_PLAN_OPERATION_0 UNIQUE ( API_TYPE_ID ) ,
	CONSTRAINT PK_TEST_PLAN_OPERATION_1 UNIQUE ( CRUD_OPERATION_TYPE_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_PLAN_OPERATION" not found; SQL statement:
COMMENT ON TABLE VSTRIKERDB.TEST_PLAN_OPERATION IS 'This table contains the test Plan operations to be executed by the test plan. ' [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "ACCOUNT" already exists; SQL statement:
CREATE TABLE VSTRIKERDB.ACCOUNT ( 
	ACCOUNT_ID           identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	NAME                 varchar(100)   NOT NULL,
	ACCOUNT_LOCATION     varchar(255)   NOT NULL,
	VALIDATED            boolean  DEFAULT 'False' NOT NULL,
	LAST_VALIDATION_DATE date   ,
	CONSTRAINT PK_ACCOUNT PRIMARY KEY ( ACCOUNT_ID )
 ) [42101-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CONFIGURATION_TEMPLATE" already exists; SQL statement:
CREATE TABLE VSTRIKERDB.CONFIGURATION_TEMPLATE ( 
	CONFIGURATION_TEMPLATE_ID identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	CONF_TEMP_OBJECT_SIZE_REPORT_UNIT_ID integer   NOT NULL,
	CONF_TEMP_NAME       varchar(255)   NOT NULL,
	CONF_TEMP_DESCRIPTION varchar(255)   ,
	CONF_TEMP_OBJECT_SIZE integer   ,
	CONF_TEMP_NUMBER_OF_OPERATIONS integer   NOT NULL,
	CONF_TEMP_NUMBER_OF_THREADS integer  DEFAULT 1 NOT NULL,
	CONF_TEMP_CREATE_OPERATION boolean   ,
	CONF_TEMP_READ_OPERATION boolean   ,
	CONF_TEMP_UPDATE_OPERATION boolean   ,
	CONF_TEMP_DELETE_OPERATION boolean   ,
	CONF_TEMP_CREATE_PERCENT integer   ,
	CONF_TEMP_READ_PERCENT integer   ,
	CONF_TEMP_UPDATE_PERCENT integer   ,
	CONF_TEMP_DELETE_PERCENT integer   ,
	CONF_TEMP_NUMBER_OF_RETRY integer   ,
	CONF_TEMP_ENABLE_RAND_SIZE_OBJECT boolean   ,
	CONF_TEMP_MIN_RAND_SIZE_OBJECT integer   ,
	CONF_TEMP_MAX_RAND_SIZE_OBJECT integer   ,
	CONF_TEMP_CREATED_DATE date  DEFAULT CURRENT_DATE ,
	CONF_TEMP_SMALL_SIZE_PERCENT integer   ,
	CONF_TEMP_MEDIUM_SIZE_PERCENT integer   ,
	CONF_TEMP_LARGE_SIZE_PERCENT integer   ,
	CONSTRAINT PK_CONFIGURATIONTEMPLATE PRIMARY KEY ( CONFIGURATION_TEMPLATE_ID ),
	CONSTRAINT PK_CONFIGURATION_TEMPLATE UNIQUE ( CONF_TEMP_OBJECT_SIZE_REPORT_UNIT_ID ) 
 ) [42101-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CRUD_OPERATION_TYPE" already exists; SQL statement:
CREATE TABLE VSTRIKERDB.CRUD_OPERATION_TYPE ( 
	CRUD_OPERATION_TYPE_ID identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	CRUD_OPERATION_NAME  varchar(100)   ,
	"CRUD OPERATION VALUE" varchar(1000)   ,
	CONSTRAINT PK_CRUD_OPERATION_TYPE PRIMARY KEY ( CRUD_OPERATION_TYPE_ID ),
	CONSTRAINT PK_CRUD_OPERATION_TYPE_0 UNIQUE ( API_TYPE_ID ) 
 ) [42101-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_REPORT" already exists; SQL statement:
CREATE TABLE VSTRIKERDB.EXECUTION_REPORT ( 
	EXECUTION_REPORT_ID  identity GENERATED ALWAYS AS IDENTITY  NOT NULL,
	EXECUTION_NAME       varchar(255)   NOT NULL,
	EXECUTION_DATE       date   ,
	TOTAL_THROUGHPUT     varchar(100)   ,
	MAX_THROUGHPUT       varchar(100)   ,
	MIN_THROUGHPUT       varchar(100)   ,
	TOTAL_VOLUME_SENT    varchar(100)   ,
	TOTAL_VOLUME_RECEIVED varchar(100)   ,
	AVG_LATENCY_PER_CRUD_OPERATION varchar(100)   ,
	NUMBER_REQUEST_SEC   integer   ,
	PERECENT_FAILED_REQUEST integer   ,
	CONSTRAINT PK_EXECUTIONREPORT PRIMARY KEY ( EXECUTION_REPORT_ID )
 ) [42101-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "OBJECT_SIZE_REPORT_UNIT" already exists; SQL statement:
CREATE TABLE VSTRIKERDB.OBJECT_SIZE_REPORT_UNIT ( 
	OBJECT_SIZE_REPORT_UNIT_ID integer GENERATED ALWAYS AS IDENTITY  NOT NULL,
	REPORT_UNIT_NAME     varchar(100)   NOT NULL,
	REPORT_UNIT_VALUE    varchar(100)   ,
	CONSTRAINT PK_OBJECTSIZEREPORTUNIT PRIMARY KEY ( OBJECT_SIZE_REPORT_UNIT_ID )
 ) [42101-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API_TYPE" already exists; SQL statement:
CREATE TABLE VSTRIKERDB.API_TYPE ( 
	API_TYPE_ID          integer GENERATED ALWAYS AS IDENTITY  NOT NULL,
	API_TYPE_NAME        varchar(100)   NOT NULL,
	API_TYPE_URL         varchar(255)   ,
	API_TYPE_DESCRIPTION varchar(255)   ,
	CONSTRAINT PK_APITYPE PRIMARY KEY ( API_TYPE_ID )
 ) [42101-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API" not found; SQL statement:
ALTER TABLE VSTRIKERDB.ACCOUNT ADD CONSTRAINT FK_ACCOUNT_API FOREIGN KEY ( ACCOUNT_ID ) REFERENCES VSTRIKERDB.API( ACCOUNT_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_PLAN" not found; SQL statement:
ALTER TABLE VSTRIKERDB.ACCOUNT ADD CONSTRAINT FK_ACCOUNT_EXECUTIONPLAN FOREIGN KEY ( ACCOUNT_ID ) REFERENCES VSTRIKERDB.EXECUTION_PLAN( ACCOUNT_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API" not found; SQL statement:
ALTER TABLE VSTRIKERDB.API_TYPE ADD CONSTRAINT FK_API_TYPE_API_TYPE_ID FOREIGN KEY ( API_TYPE_ID ) REFERENCES VSTRIKERDB.API( API_TYPE_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "FK_API_TYPE_CRUD_OPERATION_TYPE" already exists; SQL statement:
ALTER TABLE VSTRIKERDB.API_TYPE ADD CONSTRAINT FK_API_TYPE_CRUD_OPERATION_TYPE FOREIGN KEY ( API_TYPE_ID ) REFERENCES VSTRIKERDB.CRUD_OPERATION_TYPE( API_TYPE_ID ) ON DELETE CASCADE ON UPDATE CASCADE [90045-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:98)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_PLAN_OPERATION" not found; SQL statement:
ALTER TABLE VSTRIKERDB.API_TYPE ADD CONSTRAINT FK_API_TYPE_TEST_PLAN_OPERATION FOREIGN KEY ( API_TYPE_ID ) REFERENCES VSTRIKERDB.TEST_PLAN_OPERATION( API_TYPE_ID ) ON DELETE NO ACTION ON UPDATE NO ACTION [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API_SELECTED" not found; SQL statement:
ALTER TABLE VSTRIKERDB.API_TYPE ADD CONSTRAINT FK_API_TYPE_API_SELECTED FOREIGN KEY ( API_TYPE_ID ) REFERENCES VSTRIKERDB.API_SELECTED( API_TYPE_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_PLAN" not found; SQL statement:
ALTER TABLE VSTRIKERDB.CONFIGURATION_TEMPLATE ADD CONSTRAINT FK_CONFIGURATION_TEMPLATE_EXECUTION_PLAN FOREIGN KEY ( CONFIGURATION_TEMPLATE_ID ) REFERENCES VSTRIKERDB.EXECUTION_PLAN( CONFIGURATION_TEMPLATE_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_PLAN_OPERATION" not found; SQL statement:
ALTER TABLE VSTRIKERDB.CRUD_OPERATION_TYPE ADD CONSTRAINT FK_CRUD_OPERATION_TYPE_TEST_PLAN_OPERATION FOREIGN KEY ( CRUD_OPERATION_TYPE_ID ) REFERENCES VSTRIKERDB.TEST_PLAN_OPERATION( CRUD_OPERATION_TYPE_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_REPORT_DATA" not found; SQL statement:
ALTER TABLE VSTRIKERDB.EXECUTION_REPORT ADD CONSTRAINT FK_EXECUTION_REPORT_EXECUTION_REPORT_DATA FOREIGN KEY ( EXECUTION_REPORT_ID ) REFERENCES VSTRIKERDB.EXECUTION_REPORT_DATA( EXECUTION_REPORT_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_PLAN" not found; SQL statement:
ALTER TABLE VSTRIKERDB.EXECUTION_REPORT ADD CONSTRAINT FK_EXECUTION_REPORT_EXECUTION_REPORT FOREIGN KEY ( EXECUTION_REPORT_ID ) REFERENCES VSTRIKERDB.EXECUTION_PLAN( EXECUTION_REPORT_ID ) ON DELETE NO ACTION ON UPDATE NO ACTION [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_CONFIGURATION" not found; SQL statement:
ALTER TABLE VSTRIKERDB.OBJECT_SIZE_REPORT_UNIT ADD CONSTRAINT FK_OBJECT_SIZE_REPORT_UNIT_TEST_CONFIGURATION FOREIGN KEY ( OBJECT_SIZE_REPORT_UNIT_ID ) REFERENCES VSTRIKERDB.TEST_CONFIGURATION( OBJECT_SIZE_REPORT_UNIT_ID ) ON DELETE NO ACTION ON UPDATE NO ACTION [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Constraint "FK_OBJECT_SIZE_REPORT_UNIT_CONFIGURATION_TEMPLATE" already exists; SQL statement:
ALTER TABLE VSTRIKERDB.OBJECT_SIZE_REPORT_UNIT ADD CONSTRAINT FK_OBJECT_SIZE_REPORT_UNIT_CONFIGURATION_TEMPLATE FOREIGN KEY ( OBJECT_SIZE_REPORT_UNIT_ID ) REFERENCES VSTRIKERDB.CONFIGURATION_TEMPLATE( CONF_TEMP_OBJECT_SIZE_REPORT_UNIT_ID ) ON DELETE NO ACTION ON UPDATE NO ACTION [90045-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:98)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_CONFIGURATION" not found; SQL statement:
ALTER TABLE VSTRIKERDB.TEST_CONFIGURATION ADD CONSTRAINT FK_TEST_CONFIGURATION_EXECUTION_PLAN FOREIGN KEY ( TESTCONFIGURATION_ID ) REFERENCES VSTRIKERDB.EXECUTION_PLAN( TESTCONFIGURATION_ID ) ON DELETE NO ACTION ON UPDATE NO ACTION [42102-184]
12-19 15:23:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_CONFIGURATION" not found; SQL statement:
ALTER TABLE VSTRIKERDB.TEST_CONFIGURATION ADD CONSTRAINT FK_TEST_CONFIGURATION_TEST_CONFIGURATION_ID FOREIGN KEY ( TESTCONFIGURATION_ID ) REFERENCES VSTRIKERDB.API_SELECTED( TESTCONFIGURATION_ID ) ON DELETE CASCADE ON UPDATE CASCADE [42102-184]
12-19 15:33:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DROP FROM[*] VSTRIKERDB.TABLE_0 "; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE"; SQL statement:
drop FROM VSTRIKERDB.TABLE_0 [42001-184]
12-19 15:34:30 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "TABLE_0" not found; SQL statement:
drop table table_0 [42102-184]
12-19 15:36:06 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.API ( 
	API_ID               integer GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           identity   NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	URL                  varchar(255)   NOT NULL,
	SECRET_KEY           varchar(255)   NOT NULL,
	BUCKET               varchar(255)   ,
	SUBTENANT            varchar(255)   ,
	HTTP_ADDRESS_IP      varchar(16)   NOT NULL,
	HTTPS_ADDRESS_IP     varchar(16)   ,
	HTTP_ADDRESS_PORT    varchar(5)   NOT NULL,
	HTTPS_ADDRESS_PORT   varchar(5)   ,
	CONSTRAINT PK_API PRIMARY KEY ( API_ID ),
	CONSTRAINT PK_API_0 UNIQUE ( ACCOUNT_ID ) ,
	CONSTRAINT PK_API_1 UNIQUE ( API_TYPE_ID ) 
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:54:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
CREATE TABLE VSTRIKERDB.API ( 
	API_ID               integer GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           identity   NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	URL                  varchar(255)   NOT NULL,
	SECRET_KEY           varchar(255)   NOT NULL,
	BUCKET               varchar(255)   ,
	SUBTENANT            varchar(255)   ,
	HTTP_ADDRESS_IP      varchar(16)   NOT NULL,
	HTTPS_ADDRESS_IP     varchar(16)   ,
	HTTP_ADDRESS_PORT    varchar(5)   NOT NULL,
	HTTPS_ADDRESS_PORT   varchar(5)   ,
	CONSTRAINT PK_API PRIMARY KEY ( API_ID ),
 ) [90017-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.ddl.CreateTable.setPrimaryKeyColumns(CreateTable.java:266)
	at org.h2.command.ddl.CreateTable.addConstraintCommand(CreateTable.java:84)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5759)
	at org.h2.command.Parser.parseCreate(Parser.java:4141)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:446)
	at org.h2.engine.Session.prepareCommand(Session.java:388)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
12-19 15:56:13 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE VSTRIKERDB.API 2[*] ( 
	API_ID               INTEGER GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           IDENTITY  NOT NULL,
	API_TYPE_ID          INTEGER   NOT NULL,
	URL                  VARCHAR(255)   NOT NULL,
	SECRET_KEY           VARCHAR(255)   NOT NULL,
	BUCKET               VARCHAR(255)   ,
	SUBTENANT            VARCHAR(255)   ,
	HTTP_ADDRESS_IP      VARCHAR(16)   NOT NULL,
	HTTPS_ADDRESS_IP     VARCHAR(16)   ,
	HTTP_ADDRESS_PORT    VARCHAR(5)   NOT NULL,
	HTTPS_ADDRESS_PORT   VARCHAR(5)   ,
	
 ) "; SQL statement:
CREATE TABLE VSTRIKERDB.API 2 ( 
	API_ID               integer GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           IDENTITY  NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	URL                  varchar(255)   NOT NULL,
	SECRET_KEY           varchar(255)   NOT NULL,
	BUCKET               varchar(255)   ,
	SUBTENANT            varchar(255)   ,
	HTTP_ADDRESS_IP      varchar(16)   NOT NULL,
	HTTPS_ADDRESS_IP     varchar(16)   ,
	HTTP_ADDRESS_PORT    varchar(5)   NOT NULL,
	HTTPS_ADDRESS_PORT   varchar(5)   ,
	
 ) [42000-184]
12-19 15:57:11 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DROP TABLE VSTRIKERDB.API
CREATE[*] TABLE VSTRIKERDB.API ( 
	API_ID               INTEGER GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           INTEGER   NOT NULL,
	API_TYPE_ID          INTEGER   NOT NULL,
	URL                  VARCHAR(255)   NOT NULL,
	SECRET_KEY           VARCHAR(255)   NOT NULL,
	BUCKET               VARCHAR(255)   ,
	SUBTENANT            VARCHAR(255)   ,
	HTTP_ADDRESS_IP      VARCHAR(16)   NOT NULL,
	HTTPS_ADDRESS_IP     VARCHAR(16)   ,
	HTTP_ADDRESS_PORT    VARCHAR(5)   NOT NULL,
	HTTPS_ADDRESS_PORT   VARCHAR(5)   ,
	CONSTRAINT PK_API PRIMARY KEY ( API_ID ),
	CONSTRAINT PK_API_0 UNIQUE ( ACCOUNT_ID ) ,
	CONSTRAINT PK_API_1 UNIQUE ( API_TYPE_ID ) 
 ) "; SQL statement:
Drop Table VSTRIKERDB.API
CREATE TABLE VSTRIKERDB.API ( 
	API_ID               integer GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           integer   NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	URL                  varchar(255)   NOT NULL,
	SECRET_KEY           varchar(255)   NOT NULL,
	BUCKET               varchar(255)   ,
	SUBTENANT            varchar(255)   ,
	HTTP_ADDRESS_IP      varchar(16)   NOT NULL,
	HTTPS_ADDRESS_IP     varchar(16)   ,
	HTTP_ADDRESS_PORT    varchar(5)   NOT NULL,
	HTTPS_ADDRESS_PORT   varchar(5)   ,
	CONSTRAINT PK_API PRIMARY KEY ( API_ID ),
	CONSTRAINT PK_API_0 UNIQUE ( ACCOUNT_ID ) ,
	CONSTRAINT PK_API_1 UNIQUE ( API_TYPE_ID ) 
 ) [42000-184]
12-19 15:57:14 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DROP TABLE VSTRIKERDB.API
CREATE[*] TABLE VSTRIKERDB.API ( 
	API_ID               INTEGER GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           INTEGER   NOT NULL,
	API_TYPE_ID          INTEGER   NOT NULL,
	URL                  VARCHAR(255)   NOT NULL,
	SECRET_KEY           VARCHAR(255)   NOT NULL,
	BUCKET               VARCHAR(255)   ,
	SUBTENANT            VARCHAR(255)   ,
	HTTP_ADDRESS_IP      VARCHAR(16)   NOT NULL,
	HTTPS_ADDRESS_IP     VARCHAR(16)   ,
	HTTP_ADDRESS_PORT    VARCHAR(5)   NOT NULL,
	HTTPS_ADDRESS_PORT   VARCHAR(5)   ,
	CONSTRAINT PK_API PRIMARY KEY ( API_ID ),
	CONSTRAINT PK_API_0 UNIQUE ( ACCOUNT_ID ) ,
	CONSTRAINT PK_API_1 UNIQUE ( API_TYPE_ID ) 
 ) "; SQL statement:
Drop Table VSTRIKERDB.API
CREATE TABLE VSTRIKERDB.API ( 
	API_ID               integer GENERATED ALWAYS AS IDENTITY  NOT NULL,
	ACCOUNT_ID           integer   NOT NULL,
	API_TYPE_ID          integer   NOT NULL,
	URL                  varchar(255)   NOT NULL,
	SECRET_KEY           varchar(255)   NOT NULL,
	BUCKET               varchar(255)   ,
	SUBTENANT            varchar(255)   ,
	HTTP_ADDRESS_IP      varchar(16)   NOT NULL,
	HTTPS_ADDRESS_IP     varchar(16)   ,
	HTTP_ADDRESS_PORT    varchar(5)   NOT NULL,
	HTTPS_ADDRESS_PORT   varchar(5)   ,
	CONSTRAINT PK_API PRIMARY KEY ( API_ID ),
	CONSTRAINT PK_API_0 UNIQUE ( ACCOUNT_ID ) ,
	CONSTRAINT PK_API_1 UNIQUE ( API_TYPE_ID ) 
 ) [42000-184]
12-25 11:02:18 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]" [50000-184]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2759)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:750)
	at org.h2.server.web.WebApp.test(WebApp.java:895)
	at org.h2.server.web.WebApp.process(WebApp.java:221)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]" [50000-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:170)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 23 more
12-25 14:36:12 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]" [50000-184]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2759)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.eclipse.persistence.sessions.DefaultConnector.connect(DefaultConnector.java:98)
	at org.eclipse.persistence.sessions.DatasourceLogin.connectToDatasource(DatasourceLogin.java:162)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.setOrDetectDatasource(DatabaseSessionImpl.java:204)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.loginAndDetectDatasource(DatabaseSessionImpl.java:741)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.login(EntityManagerFactoryProvider.java:239)
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:685)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:204)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:304)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:336)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:302)
	at vStrikerUnitTest.vStrikerEntitesUnitTest.TestAccountCreate(vStrikerEntitesUnitTest.java:38)
	at vStrikerUnitTest.vStrikerEntitesUnitTest.test(vStrikerEntitesUnitTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]" [50000-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 58 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:170)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 53 more
12-25 14:36:12 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]" [50000-184]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2759)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at org.eclipse.persistence.sessions.DefaultConnector.directConnect(DefaultConnector.java:165)
	at org.eclipse.persistence.sessions.DefaultConnector.connect(DefaultConnector.java:117)
	at org.eclipse.persistence.sessions.DatasourceLogin.connectToDatasource(DatasourceLogin.java:162)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.setOrDetectDatasource(DatabaseSessionImpl.java:204)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.loginAndDetectDatasource(DatabaseSessionImpl.java:741)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.login(EntityManagerFactoryProvider.java:239)
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:685)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:204)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:304)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:336)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:302)
	at vStrikerUnitTest.vStrikerEntitesUnitTest.TestAccountCreate(vStrikerEntitesUnitTest.java:38)
	at vStrikerUnitTest.vStrikerEntitesUnitTest.test(vStrikerEntitesUnitTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]" [50000-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 57 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:170)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 52 more
12-25 14:43:46 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]" [50000-184]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2759)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.eclipse.persistence.sessions.DefaultConnector.connect(DefaultConnector.java:98)
	at org.eclipse.persistence.sessions.DatasourceLogin.connectToDatasource(DatasourceLogin.java:162)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.setOrDetectDatasource(DatabaseSessionImpl.java:204)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.loginAndDetectDatasource(DatabaseSessionImpl.java:741)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.login(EntityManagerFactoryProvider.java:239)
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:685)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:204)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:304)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:336)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:302)
	at vStrikerUnitTest.vStrikerEntitesUnitTest.TestAccountCreate(vStrikerEntitesUnitTest.java:38)
	at vStrikerUnitTest.vStrikerEntitesUnitTest.test(vStrikerEntitesUnitTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]" [50000-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 58 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:170)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 53 more
12-25 14:43:47 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]" [50000-184]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2759)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at org.eclipse.persistence.sessions.DefaultConnector.directConnect(DefaultConnector.java:165)
	at org.eclipse.persistence.sessions.DefaultConnector.connect(DefaultConnector.java:117)
	at org.eclipse.persistence.sessions.DatasourceLogin.connectToDatasource(DatasourceLogin.java:162)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.setOrDetectDatasource(DatabaseSessionImpl.java:204)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.loginAndDetectDatasource(DatabaseSessionImpl.java:741)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.login(EntityManagerFactoryProvider.java:239)
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:685)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:204)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:304)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:336)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:302)
	at vStrikerUnitTest.vStrikerEntitesUnitTest.TestAccountCreate(vStrikerEntitesUnitTest.java:38)
	at vStrikerUnitTest.vStrikerEntitesUnitTest.test(vStrikerEntitesUnitTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]" [50000-184]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 57 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/EMC/emccode/vstriker/vStrikerDB/DB/vstrikerdb.mv.db [1.4.184/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:170)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 52 more
12-25 14:50:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "ACCOUNT" not found; SQL statement:
INSERT INTO ACCOUNT (ACCOUNT_LOCATION, LAST_VALIDATION_DATE, NAME, VALIDATED) VALUES (?, ?, ?, ?) [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "ACCOUNT" not found; SQL statement:
SELECT ACCOUNT_ID FROM VSTRIKERDB.ACCOUNT WHERE ACCOUNT_ID <> ACCOUNT_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API" not found; SQL statement:
SELECT API_ID FROM VSTRIKERDB.API WHERE API_ID <> API_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API_SELECTED" not found; SQL statement:
SELECT API_SELECTED_ID FROM VSTRIKERDB.API_SELECTED WHERE API_SELECTED_ID <> API_SELECTED_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API_TYPE" not found; SQL statement:
SELECT API_TYPE_ID FROM VSTRIKERDB.API_TYPE WHERE API_TYPE_ID <> API_TYPE_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CONFIGURATION_TEMPLATE" not found; SQL statement:
SELECT CONFIGURATION_TEMPLATE_ID FROM VSTRIKERDB.CONFIGURATION_TEMPLATE WHERE CONFIGURATION_TEMPLATE_ID <> CONFIGURATION_TEMPLATE_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CRUD_OPERATION_TYPE" not found; SQL statement:
SELECT CRUD_OPERATION_TYPE_ID FROM VSTRIKERDB.CRUD_OPERATION_TYPE WHERE CRUD_OPERATION_TYPE_ID <> CRUD_OPERATION_TYPE_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_PLAN" not found; SQL statement:
SELECT EXECUTION_PLAN_ID FROM VSTRIKERDB.EXECUTION_PLAN WHERE EXECUTION_PLAN_ID <> EXECUTION_PLAN_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_REPORT" not found; SQL statement:
SELECT EXECUTION_REPORT_ID FROM VSTRIKERDB.EXECUTION_REPORT WHERE EXECUTION_REPORT_ID <> EXECUTION_REPORT_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_REPORT_DATA" not found; SQL statement:
SELECT EXECUTION_REPORT_DATA_ID FROM VSTRIKERDB.EXECUTION_REPORT_DATA WHERE EXECUTION_REPORT_DATA_ID <> EXECUTION_REPORT_DATA_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "OBJECT_SIZE_REPORT_UNIT" not found; SQL statement:
SELECT OBJECT_SIZE_REPORT_UNIT_ID FROM VSTRIKERDB.OBJECT_SIZE_REPORT_UNIT WHERE OBJECT_SIZE_REPORT_UNIT_ID <> OBJECT_SIZE_REPORT_UNIT_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_CONFIGURATION" not found; SQL statement:
SELECT TESTCONFIGURATION_ID FROM VSTRIKERDB.TEST_CONFIGURATION WHERE TESTCONFIGURATION_ID <> TESTCONFIGURATION_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_PLAN_OPERATION" not found; SQL statement:
SELECT TEST_PLAN_OPERATION_ID FROM VSTRIKERDB.TEST_PLAN_OPERATION WHERE TEST_PLAN_OPERATION_ID <> TEST_PLAN_OPERATION_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "ACCOUNT" not found; SQL statement:
SELECT ACCOUNT_ID FROM VSTRIKERDB.ACCOUNT WHERE ACCOUNT_ID <> ACCOUNT_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API" not found; SQL statement:
SELECT API_ID FROM VSTRIKERDB.API WHERE API_ID <> API_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API_SELECTED" not found; SQL statement:
SELECT API_SELECTED_ID FROM VSTRIKERDB.API_SELECTED WHERE API_SELECTED_ID <> API_SELECTED_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "API_TYPE" not found; SQL statement:
SELECT API_TYPE_ID FROM VSTRIKERDB.API_TYPE WHERE API_TYPE_ID <> API_TYPE_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CONFIGURATION_TEMPLATE" not found; SQL statement:
SELECT CONFIGURATION_TEMPLATE_ID FROM VSTRIKERDB.CONFIGURATION_TEMPLATE WHERE CONFIGURATION_TEMPLATE_ID <> CONFIGURATION_TEMPLATE_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CRUD_OPERATION_TYPE" not found; SQL statement:
SELECT CRUD_OPERATION_TYPE_ID FROM VSTRIKERDB.CRUD_OPERATION_TYPE WHERE CRUD_OPERATION_TYPE_ID <> CRUD_OPERATION_TYPE_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_PLAN" not found; SQL statement:
SELECT EXECUTION_PLAN_ID FROM VSTRIKERDB.EXECUTION_PLAN WHERE EXECUTION_PLAN_ID <> EXECUTION_PLAN_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_REPORT" not found; SQL statement:
SELECT EXECUTION_REPORT_ID FROM VSTRIKERDB.EXECUTION_REPORT WHERE EXECUTION_REPORT_ID <> EXECUTION_REPORT_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "EXECUTION_REPORT_DATA" not found; SQL statement:
SELECT EXECUTION_REPORT_DATA_ID FROM VSTRIKERDB.EXECUTION_REPORT_DATA WHERE EXECUTION_REPORT_DATA_ID <> EXECUTION_REPORT_DATA_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "OBJECT_SIZE_REPORT_UNIT" not found; SQL statement:
SELECT OBJECT_SIZE_REPORT_UNIT_ID FROM VSTRIKERDB.OBJECT_SIZE_REPORT_UNIT WHERE OBJECT_SIZE_REPORT_UNIT_ID <> OBJECT_SIZE_REPORT_UNIT_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_CONFIGURATION" not found; SQL statement:
SELECT TESTCONFIGURATION_ID FROM VSTRIKERDB.TEST_CONFIGURATION WHERE TESTCONFIGURATION_ID <> TESTCONFIGURATION_ID [42102-184]
12-26 17:41:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TEST_PLAN_OPERATION" not found; SQL statement:
SELECT TEST_PLAN_OPERATION_ID FROM VSTRIKERDB.TEST_PLAN_OPERATION WHERE TEST_PLAN_OPERATION_ID <> TEST_PLAN_OPERATION_ID [42102-184]
